template/cordova/lib/MSBuildTools.js

```diff
@@ -38,6 +38,13 @@ MSBuildTools.prototype.buildProject = function(projFile, buildType, buildarch) {
     '/p:Configuration=' + buildType,
     '/p:Platform=' + buildarch];
 
+    if (otherConfigProperties) {
```

Comment:
I'll look into the .targets approach.

However, I don't agree with the IDE-CLI interop dichotomy.  In the IDE, you almost always are building a single arch+mode target pair at a time (e.g., x86-debug).  In VS, you can generate app package bundles, but you have to go through a particular menu to do so, not use the typical MSBuild-invoking "Build" step workflow.  The same works here as well.

The change was designed 100% to be just like the "Create App Packages" feature, just turned on by default when building arch-specific packages.